.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '18 23 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue as been marked as stale, if you require further assistance please'
+        stale-pr-message: 'Stale pull request message'
+        stale-issue-label: 'Stale'
```

Comment:
We should add the 'exempt-issue-labels' and 'exempt-pr-labels' flag so we can apply labels to issue's/pull requests we don't want labeled as stale/closed

Reply:
agree